com.unity.multiplayer.mlapi/Runtime/Core/NetworkingManager.cs

```diff
@@ -47,13 +47,30 @@ public class NetworkingManager : MonoBehaviour
         static ProfilerMarker s_EventTick = new ProfilerMarker("Event");
         static ProfilerMarker s_ReceiveTick = new ProfilerMarker("Receive");
         static ProfilerMarker s_SyncTime = new ProfilerMarker("SyncTime");
-        static ProfilerMarker s_TransportConnect =
-            new ProfilerMarker("TransportConnect");
-        static ProfilerMarker s_HandleIncomingData =
-            new ProfilerMarker("HandleIncomingData");
-        static ProfilerMarker s_TransportDisconnect =
-            new ProfilerMarker("TransportDisconnect");
+        static ProfilerMarker s_TransportConnect = new ProfilerMarker("TransportConnect");
+        static ProfilerMarker s_HandleIncomingData = new ProfilerMarker("HandleIncomingData");
+        static ProfilerMarker s_TransportDisconnect = new ProfilerMarker("TransportDisconnect");
+
+
+        static ProfilerMarker s_MLAPIServerRPC = new ProfilerMarker("MLAPIServerRPC");
+        static ProfilerMarker s_MLAPIServerRPCQueued = new ProfilerMarker("MLAPIServerRPCQueued");
+
+        static ProfilerMarker s_MLAPIClientRPC = new ProfilerMarker("MLAPIClientRPC");
+        static ProfilerMarker s_MLAPIClientRPCQueued = new ProfilerMarker("MLAPIClientRPCQueued");
+
+        static ProfilerMarker s_MLAPIServerSTDRPC = new ProfilerMarker("MLAPIServerSTDRPC");
+        static ProfilerMarker s_MLAPIServerSTDRPCQueued = new ProfilerMarker("MLAPIServerSTDRPCQueued");
+
+        static ProfilerMarker s_MLAPIClientSTDRPC = new ProfilerMarker("MLAPIClientSTDRPC");
+        static ProfilerMarker s_MLAPIClientSTDRPCQueued = new ProfilerMarker("MLAPIClientSTDRPCQueued");
+        static ProfilerMarker s_InvokeRPC = new ProfilerMarker("InvokeRPC");
+
+
 #endif
+        [HideInInspector]
+        public bool LoopbackEnabled;
```

Comment:
wouldn't "loopback" be an implicit implementation details where we just put RPC into inbound queue without necessarily checking if we "enabled" loopbacks? (maybe I should look at implementation details a little bit more but I feel like loopback could be just an implementation detail behind `BeginSendServerRpc`+`EndSendServerRpc` combo when it's `IsHost == true` and it goes into inbound queue rather than outbound queue)

Reply:
Since this is primarily used as a tool, I wanted to expose it at the editor level.  However, I am open to any alternate methods in checking for this scenario (and/or potentially adding additional code to just redirect messages to a Host client).